python/src/trezorlib/cardano.py

```diff
@@ -128,6 +130,44 @@ def _create_change_output(output) -> messages.CardanoTxOutputType:
     )
 
 
+def create_certificate(certificate) -> messages.CardanoTxCertificateType:
+    if not all(certificate.get(k) is not None for k in REQUIRED_FIELDS_CERTIFICATE):
+        raise ValueError("The certificate is missing some fields")
+
+    path = certificate["path"]
+    certificate_type = certificate["type"]
+
+    if certificate_type == messages.CardanoCertificateType.STAKE_DELEGATION:
+        if certificate.get("pool") is None:
+            raise ValueError("The certificate is missing some fields")
+
+        pool = certificate["pool"]
+        return messages.CardanoTxCertificateType(
+            type=certificate["type"],
```

Comment:
be consistent -- either you should use certificate["type"] or certificate_type but do not mix it in the code

Reply:
Definitely, thanks. Fixed.